source/reference/supported-server-version.txt

```diff
@@ -17,23 +17,50 @@ version limitations and requirements:
 
 - .. include:: /includes/fact-no-8.0-support.rst
 
-- .. include:: /includes/fact-minimum-server-version-support.rst
-
 - ``mongosync`` doesn't support MongoDB rapid releases, such as 6.1 or 6.2.
 
-- .. include:: /includes/fact-minimum-fcv.rst 
+- Your source cluster must run one of the following major supported versions: 
+  
+  - 4.4 (*New in version 1.10.0*)
+  - 5.0 (*New in version 1.10.0*)
+  - 6.0
+  - 7.0
+
+The following table lists the minimum MongoDB server version requirements for  
+source clusters:
+
+.. list-table::
+   :header-rows: 1
+   :class: compatibility 
+
+   * - Major Version 
+     - Patch Version 
+     - Minimum Feature Compatibility Version
+
+   * - 4.4
+     - 4.4.23
+     - 4.4
 
-Synchronize Data Between Clusters Running Older MongoDB Server Versions
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+   * - 5.0
+     - 5.0.29
+     - 5.0
 
-.. include:: /includes/fact-minimum-versions.rst
+   * - 6.0
+     - 6.0.16
+     - 6.0
 
+   * - 7.0
+     - 7.0.9
+     - 6.0
 
 Synchronize Data Between Clusters with Different MongoDB Server Major Versions
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+------------------------------------------------------------------------------
 
 .. include:: /includes/cross-version-sync.rst
 
+Starting in version 1.10.0, ``mongosync`` supports migrations from pre-6.0 
```

Comment:
I think it would be helpful to add an explicit note that mongosync does not support pre-6.0 destination clusters

Reply:
added